MediaBrowser.Controller/Net/IWebSocketConnection.cs

```diff
@@ -26,6 +26,12 @@ public interface IWebSocketConnection : IDisposable
         /// <value>The last activity date.</value>
         DateTime LastActivityDate { get; }
 
+        /// <summary>
+        /// Gets or sets the date of last Keeplive received.
+        /// </summary>
+        /// <value>The date of last Keeplive received.</value>
+        public DateTime LastKeepAliveDate { get; set; }
```

Comment:
```suggestion
        DateTime LastKeepAliveDate { get; set; }
```
Too stay consistent with other interface members